### DISCUSSION TITLE
One of, for zod strings

### DISCUSSION THREAD

**Discussion by Ellathet** [colinhacks/zod#1891] [Q&A] [answered]
Using zod, I missed a property of `oneOf`, like have in yup, for example:
```js
yup.string().oneOf(['horse', 'apple'])
```
In zod, is there any similar property to use in a string?

**Comment by JacobWeisenburger** [comment, marked as answer]
Are these what you're looking for?
https://github.com/colinhacks/zod#zod-enums
https://github.com/colinhacks/zod#literals

**Reply by Ellathet** [reply]
Yes is this, thank you soo much!
